reaper/reaper.go

```diff
@@ -266,13 +303,20 @@ func getInstances() chan *reaperaws.Instance {
 			// make the map if it is not initialized
 			if instanceTypeSums[instance.Region] == nil {
 				instanceTypeSums[instance.Region] = make(map[string]int)
+				if instance.InstanceType != nil {
+					// increment InstanceType counter
+					instanceTypeSums[instance.Region][*instance.InstanceType]++
+				}
 			}
-			instanceTypeSums[instance.Region][instance.InstanceType]++
 
 			regionSums[instance.Region]++
 			ch <- instance
 		}
 
+		for region, sum := range regionSums {
```

Comment:
Log # of resources outside of events so that it happens if there are no EventReporters